pandas/core/frame.py

```diff
@@ -7840,6 +7842,12 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
         C        1 days 12:00:00
         Name: 0.5, dtype: object
         """
+        if isinstance(q, int) and q > 1:
+            return self.quantile(q=[i / q for i in range(1, q)],
```

Comment:
i am not sure this that intuitive 
why is it difficult to use linspace(0, 1, 10) here? would take that as an example

Reply:
`linspace(0, 1, 10)` will include the min and max as well (which I think people often won't want when they're looking for quantiles) and so we actually need `linspace(0, 1, 11)`, which is a little unusual.  I was thinking that `quantile(10)` would be an easy way to get the cuts themselves for dividing the data into 10 equally-sized groups.